DV permission component stories

mdrillin opened the pull request:
changes related to TEIIDTOOLS-979

- Adds stories for a few of the permission components
- other minor DV story updates
- a few i18n additions
- minor layout adjustments

indraraj commented on app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json:
```diff
@@ -107,6 +107,8 @@
   "permissionSelectAll": "Select all ({{allListLength}} items)",
   "permissionSet": "Set permissions",
   "permissionSeletedViews": "Grant or Revoke permissions for the selected Views: ",
+  "permissionShowLess": "Show less",
```
these can be moved to shared and can remove permission from the key.

mdrillin replied:
agree - there are probably others that could be in shared too

indraraj commented:
not sure why we removed withNotes

mdrillin commented:
> not sure why we removed withNotes

They were not showing up in storybook.  They were not removed, just changed the implementation which needed updating.
